pom.xml

```diff
@@ -521,12 +521,12 @@
 			<dependency>
 				<groupId>org.aspectj</groupId>
 				<artifactId>aspectjrt</artifactId>
-				<version>1.9.4</version>
+				<version>1.9.5</version>
 			</dependency>
 			<dependency>
 				<groupId>org.aspectj</groupId>
 				<artifactId>aspectjweaver</artifactId>
-				<version>1.9.4</version>
+				<version>1.9.5</version>
```

Comment:
try out the changes with reference application

Reply:
thanks @HerbertYiga I tested it in the same way as all the other PRs that I perform, by executing the jetty: run command, with the legacyUI module install.

Reply:
hello @HerbertYiga , i tested with the standalone reference application.
![Screenshot from 2020-06-16 11-46-44](https://user-images.githubusercontent.com/39851246/84765846-b946d080-afc7-11ea-8e2e-d204fefcb3b7.png)